src/Room/Room.js

```diff
@@ -1,41 +1,77 @@
-import { StatusBar } from "expo-status-bar";
-import React from "react";
-import { StyleSheet, View, Image } from "react-native";
-import { Button } from "react-native-paper";
+import React, { useState, useEffect } from "react";
+import { StyleSheet, View, Text, Image, TouchableOpacity } from "react-native";
+import { getRoom, getMemories } from "../apiCalls";
+import  Memory  from "../Memory/Memory";
 
-export default Room = (props) => {
-  // here's how you'll access the id
-  console.log(props.route.params.id);
+export default function Room(props) {
+  const selectedRoom = props.route.params.id;
 
-  const buttonPress = () => {
-    props.navigation.navigate("Memory");
+  const [room, setRoom] = useState({});
+  const [memories, setMemories] = useState([]);
+   const [memoryIsSelected, setMemoryAsSelected] = useState(false);
+
+  useEffect(() => {
+    getSelectedRoom();
+    getRoomMemories();
+  }, []);
+
+  const getSelectedRoom = async () => {
+    await getRoom(selectedRoom)
+      .then((data) => setRoom(data.data))
+      .then(() => {
+      })
+
+      .catch((error) => console.error(error));
+  };
+
+  const getRoomMemories = async () => {
+    await getMemories(selectedRoom)
+      .then((data) => setMemories(data.data))
+      .then((data) => console.log(memories))
+      .catch((error) => console.error(error));
   };
 
   return (
-    <View style={styles.container}>
-      <Button
+    <TouchableOpacity
+      onPress={(event) =>
+        console.log(event.nativeEvent.locationX, event.nativeEvent.locationY)
+      }
+      style={styles.container}
+    >
+      <Text>Hopefully a room appears</Text>
+      <Image
+        source={{ uri: `${room.image}` }}
+        style={{ width: 820, height: 1180 }}
+      />
+      <TouchableOpacity
+        style={styles.button}
         onPress={() => {
-          buttonPress();
+          setMemoryAsSelected(true);
         }}
-        style={{ margin: 20 }}
       >
-        TO MEMORY
-      </Button>
-      <Image
-        source={require("../../assets/backdrops/livingroom.png")}
-        style={{ width: "90%", height: "90%" }}
-      ></Image>
-      <StatusBar style="auto" />
-    </View>
+        {memoryIsSelected && <Memory memory={memories[0]} />}
+      </TouchableOpacity>
+    </TouchableOpacity>
   );
-};
+}
 
 const styles = StyleSheet.create({
   container: {
+    zIndex: -1,
     display: "flex",
     backgroundColor: "#fff",
     fontWeight: "bold",
     alignItems: "center",
     justifyContent: "center",
   },
+  button: {
+    zIndex: 2,
+    position: "absolute",
+    borderWidth: 4,
+    borderColor: "red",
+    bottom: 200,
+    left: 60,
```

Comment:
These two values are what we are making dynamic to change with the x and y coordinates that are coming in with each memory.